src/Microsoft.AspNetCore.Mvc.Abstractions/Abstractions/IActionInvokerProvider.cs

```diff
@@ -26,8 +47,16 @@ public interface IActionInvokerProvider
         /// </remarks>
         int Order { get; }
 
+        /// <summary>
+        /// Called to execute the provider. 
+        /// </summary>
+        /// <param name="context">The <see cref="ActionInvokerProviderContext"/>.</param>
         void OnProvidersExecuting(ActionInvokerProviderContext context);
 
+        /// <summary>
+        /// Called to execute the provider, after all <see cref="OnProvidersExecuting"/> methods have been called.
```

Comment:
Alternate Suggestion (Feel free to ignore): 
`Called to execute the provider, after the <see cref="OnProvidersExecuting"/> method of all <see cref="IActionInvokerProvider"/> have been called.`